qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -45,7 +45,9 @@ class QSVC(SVC, SerializableModelMixin):
     def __init__(self, *, quantum_kernel: Optional[BaseKernel] = None, **kwargs):
         """
         Args:
-            quantum_kernel: Quantum kernel to be used for classification.
+            quantum_kernel: A subclass of :class:`~qiskit_machine_learning.kernels.BaseKernel`
+                to be used for classification.
+                If None, default to :class:`~qiskit_machine_learning.kernels.FidelityQuantumKernel`.
```

Comment:
This sentence could also be added to improve PegasosQSVC with the additional note that this is only the case when precomputed is False.